src/lang/en.json

```diff
@@ -736,8 +736,8 @@
     "Badge Type": "Badge Type",
     "Badge Duration": "Badge Duration",
     "Badge Label": "Badge Label",
-    "Badge Prefix": "Badge Prefix",
-    "Badge Suffix": "Badge Suffix",
+    "Badge Prefix": "Badge Value Prefix",
+    "Badge Suffix": "Badge Value Suffix",
```

Comment:
For me, this is not clearer with the added value.
Imo, the current translation is good.

```suggestion
    "Badge Prefix": "Badge Prefix",
    "Badge Suffix": "Badge Suffix",
```

Reply:
"Badge Prefix" literally something in front of the badge. But that's not what it does.

![image](https://github.com/louislam/uptime-kuma/assets/3271800/d8502593-db2c-4238-a8b3-c5bda179be49)

![image](https://github.com/louislam/uptime-kuma/assets/3271800/fabeae9c-1343-42ba-ba15-c9a833ed0ea4)

"Badge Value Prefix" describes that the prefix is for the value of the badge.